audio-echo/app/build.gradle

```diff
@@ -6,22 +6,18 @@ android {
     defaultConfig {
         applicationId 'com.google.sample.echo'
         /*
-           To run on earlier version of Android than android-21, do:
+           To run on earlier version of Android other than android-23, do:
```

Comment:
Why is the `minSdkVersion` set so high if it doesn't need to be?

Reply:
23 should be the one beginning dynamic run-time permission request.

Reply:
But that's `targetSdkVersion`, not `minSdkVersion`.